charts/consul/Chart.yaml

```diff
@@ -13,7 +13,7 @@ annotations:
   artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.13.2
+      image: hashicorp/consul:1.13.3
     - name: consul-k8s-control-plane
       image: hashicorp/consul-k8s-control-plane:0.49.0
```

Comment:
Shouldn't this be 0.49.1 as well?

Reply:
Shouldn't this be done by `make prepare-release`?

Reply:
i think the make prepare-release step does this

Reply:
Oh maybe there's a separate PR for the release @ndhanushkodi ?

Reply:
the steps are a bit confusing https://github.com/hashicorp/engineering-docs/blob/main/consul/consul-k8s/release-process.md

it says to not include the make prepare-release changes in the PR but I think we have to include them. I've included both sets of changes here now.

Reply:
I think the instructions just meant go ahead and push the changes direct, no need to open a PR when creating the branch for the first time

Reply:
I think you can add an addendum to a step that says open a PR if this is a patch on the original code freeze/release